### DISCUSSION TITLE
reset function call changes in all controllers of page

### DISCUSSION THREAD

**Discussion by lucasgomesdc** [react-hook-form/react-hook-form#12061] [General]
Hi everyone,

I have a problem with use reset to clean my form.
When I use that function, my form that have two selects wrapped by Controller, even that I define default value with only one, both did clean.

Can I have way to prevent this?
Example of code:

```tsx
reset({
  state: ''
});

return (
  <>
    <Controller 
      name='country'
      render={({ field }) => <Select {...field} />}
    />
    <Controller 
      name='state'
      render={({ field }) => <Select {...field} />}
    />
  </>
);

```

After reset state is '' and country is undefined.

**Comment by tatsmaki** [comment]
Did you use ?
```
useForm({
    defaultValues: {
        country: '',
        state: ''
    }
})
